fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/QuerySegmentAggregator.java

```diff
@@ -604,7 +606,10 @@ protected void buildWhereClause(StringBuilder whereClause, String overrideType)
                         .append(".LOGICAL_RESOURCE_ID = R.LOGICAL_RESOURCE_ID");
                     }
                 }
-            } // end if SKIP_WHERE
+            } else {
+                // end if SKIP_WHERE
+                System.out.println(whereClause);
```

Comment:
did you mean to take this back out?  if not, it should be a log statement, not sysout

Reply:
Yes, I meant to remove.  Fixed in next commit.